README.md

```diff
@@ -0,0 +1,111 @@
+Substrait Consumer Integration Tests
+====================================
+
+Table of Contents
+=================
+* [Overview](#Overview)
+* [Setup](#Setup)
+* [How to Run Tests](#How-to-Run-Tests)
+* [Test Files](#Test-Files)
+  * [Test Case Args](#Test-Case-Args)
+  * [Substrait Plans](#Substrait-Plans)
+  * [SQL Queries](#SQL-Queries)
+* [How to Add New Consumers](#How-to-Add-New-Consumers)
+
+# Overview
+This testing repository provides instructions on how to add and run substrait consumer 
+integration tests.  These tests take in parquet files, a SQL query, and a json-formatted substrait 
+query plan as input.  The tests will run the SQL query on DuckDB, and the substrait plan on the 
+specified consumer, and then compare the two results. The test data is created using DuckDB at the 
+start of the test class using the `prepare_tpch_parquet_data` fixture, which is located in 
+`tests/conftest.py`.
+
+# Setup
+
```

Comment:
```suggestion
```
Minor nit for consistency (alternatively, you could add a blank line after the other # headers)